docker-compose.yaml

```diff
@@ -1,28 +1,67 @@
 version: '3'
 
+# Generate self-signed certs with:
+# openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes
+# openssl req -new -key key.pem -x509 -days 1000 -out CA-cert.pem
+
+# # Generating certificates: https://opendistro.github.io/for-elasticsearch-docs/docs/security-configuration/generate-certificates/
+# # First two commands result in root CA file: root-ca.pem
+# openssl genrsa -out root-ca-key.pem 2048
+# openssl req -new -x509 -sha256 -key root-ca-key.pem -out root-ca.pem
+# # Next few commands generate key (admin-key.pem) and certificate (admin.pem)
+# openssl genrsa -out admin-key-temp.pem 2048
+# openssl pkcs8 -inform PEM -outform PEM -in admin-key-temp.pem -topk8 -nocrypt -v1 PBE-SHA1-3DES -out admin-key.pem
+# openssl req -new -key admin-key.pem -out admin.csr
+# openssl x509 -req -in admin.csr -CA root-ca.pem -CAkey root-ca-key.pem -CAcreateserial -sha256 -out admin.pem
+# #...
+# #Then export JAVA_HOME=/usr/share/elasticsearch/jdk
```

Comment:
Is the whole point of this commented out section that contains commands to show what is happening with the docker-compose command, or is it to show how someone might do it manually?

Reply:
That is a note about setting  up the certificates manually. I've since added that to the README.md so this can probably be removed from the docker-compose file.

Reply:
I noticed the same as @cookersjs - has or can this section of comments be removed?

Reply:
This is cleaned up now.